.github/workflows/ci.yml

```diff
@@ -541,7 +541,14 @@ jobs:
           && ! (contains(matrix.modules, 'trino-filesystem-gcs') && contains(matrix.profile, 'cloud-tests'))
           && ! (contains(matrix.modules, 'trino-filesystem-s3') && contains(matrix.profile, 'cloud-tests'))
           && ! (contains(matrix.modules, 'trino-hdfs') && contains(matrix.profile, 'cloud-tests'))
-        run: $MAVEN test ${MAVEN_TEST} -pl ${{ matrix.modules }} ${{ matrix.profile != '' && format('-P {0}', matrix.profile) || '' }}
+        run: |
+          if [ "${{ matrix.modules }}" == "plugin/trino-oracle" ]; then
+            echo "Running tests for oracle..."
+            $MAVEN test ${MAVEN_TEST} -pl ${{ matrix.modules }} ${{ matrix.profile != '' && format('-P {0}', matrix.profile) || '' }} 2>&1 | grep "ERROR" || true
+          else
+            echo "Running tests for ${{ matrix.modules }}..."
+            $MAVEN test ${MAVEN_TEST} -pl ${{ matrix.modules }} ${{ matrix.profile != '' && format('-P {0}', matrix.profile) || '' }}
+          fi
```

Comment:
Can we revert this change?

Reply:
@ebyhr Let me do more round tests

Reply:
@ebyhr this change is for reduce the logs of the oracle tests, which is required of this pr.

Could you please see the stress test results? I just triggle tests of oracle, there is one successful and others failed at uploading the test result phrase.

Reply:
> this change is for reduce the logs of the oracle tests, which is required of this pr.

Oh, I thought it's just for stress tests. Is it difficult to reduce logs within Oralce tests instead of ci.yml?

Reply:
This is a question actually I also want to ask...
cc @wendigo @hashhar do you happen to have any idea how to let the whole test only log of error level

Reply:
Which loggers are chatty?

Reply:
@wendigo Just in case you wanna see the failure about `No space left`, may run multi oracle tests together, like stress test do.